runtime/include/chpl-mem.h

```diff
@@ -249,14 +249,14 @@ size_t chpl_mem_localizationThreshold(void) {
 
 #else // LAUNCHER
 
-#include <stdlib.h>
+#include "chpl-mem-sys.h"
```

Comment:
The check-for-malloc script is a glorified grep (does a fuzzy parse of the source code using cscope) so there's no simple way of doing this, and I personally like switching the launcher case over to sys_{malloc, free}